src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -33,71 +42,106 @@
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  * @author Saša Stamenković <umpirsky@gmail.com>
  */
-class ResourceController extends FOSRestController
+class ResourceController extends ContainerAware
```

Comment:
Why would you discourage them? The aim of ResourceController is to ease some development. Not extending Controller seems a constraint that does not solve anything (deps are still properly injected!), thus a useless constraint to me ;)